src/common-utils/main.sh

```diff
@@ -518,6 +518,7 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
     # See https://github.com/ohmyzsh/ohmyzsh/blob/master/tools/install.sh for official script.
     if [ "${INSTALL_OH_MY_ZSH}" = "true" ]; then
         user_rc_file="${user_home}/.zshrc"
+        copy_user_rc_file="${user_home}/.zshrc-copy"
```

Comment:
Isn't `user_home` a temporary file? So can we create a new temp folder and add it there? 

Once the copying over is completed, let's delete the folder

Reply:
discarding this too as copying a file is not required now